data/journal_check/bug_refs.json

```
@@ -45,6 +45,13 @@
         },
         "type": "bug"
     },
+    "bsc#1188444": {
+        "description": "[Firmware Bug]: Bootloader left irq enabled: bank 1 irq 9",
```

Needs regex escaping, but let's just avoid having to deal with that part:

```suggestion
        "description": "Bootloader left irq enabled: bank 1 irq 9",
```

---

Updated